### ISSUE TITLE
Watch files in dev mode

### ISSUE THREAD

**Issue by berndartmueller** [vinissimus/next-translate#343]
Hello,

thanks a lot for this awesome and helpful library!

I was wondering if it's possible in development mode to re-build the `pages` folder whenever a file in `_pages` (currentPagesDir) or any of it's dependencies (e.g. components) changes?

Currently, I'm using the npm script command  `dev: next-translate && next dev`, but I'm missing the file watcher.

Thanks!

**Comment by aralroca** [comment]
@berndartmueller this will be solved in 1.0.0 😊  Related issue: https://github.com/vinissimus/next-translate/issues/303 I encourage you to try the [1.0.0-experimental.14](https://github.com/vinissimus/next-translate/releases/tag/1.0.0-experimental.14) prerelease. There are a few things to finish/fix, hopefully, this will be available in December! In fact, it would be a great help if you can try the prerelease and give your feedback (reporting possible issues). Thank you very much! 🙏 😊

**Comment by berndartmueller** [comment]
@aralroca Ok, awesome! I'm looking forward to 1.0.0 👍 

In the meantime, I managed to get a working solution with [npm-watch](https://github.com/M-Zuber/npm-watch) and [concurrently](https://github.com/kimmobrunfeldt/concurrently#readme):

package.json:

```sh
...
  "watch": {
    "next-i18n": {
      "extensions": "js,jsx,ts,tsx",
      "patterns": [
        "src"
      ],
      "ignore": [
        "src/pages"
      ]
    }
  },
  "scripts": {
    "next-i18n": "next-translate",
    "dev": "concurrently \"npm-watch next-i18n\" \"next dev\"",
  },
...
```

**Comment by aralroca** [comment]
Closing this. Already fixed on [1.0.0-canary.1](https://github.com/vinissimus/next-translate/releases/tag/1.0.0-canary.1). No more need to work on `pages_`, you can directly work on `pages`. The 1.0 will be released in approximately 1-2 weeks. 😊
